modules/SofaTopologyMapping/src/SofaTopologyMapping/Edge2QuadTopologicalMapping.cpp

```diff
@@ -81,16 +81,16 @@ void Edge2QuadTopologicalMapping::init()
         msg_error() << "Radius is zero or negative";
         return;
     }
-    else if (d_radiusFocal.isSet() && d_radiusFocal.getValue() < std::numeric_limits<double>::min())
+    if (d_radiusFocal.isSet() && d_radiusFocal.getValue() < std::numeric_limits<double>::min())
     {
-        msg_error() << "Focal Radius is zero or negative";
-        return;
+        msg_warning() << "Focal Radius is zero or negative";
     }
+
     double rho = d_radius.getValue();
 
     bool ellipse = false;
     double rhoFocal;
-    if (d_radiusFocal.isSet())
+    if (d_radiusFocal.isSet() && d_radiusFocal.getValue() >= std::numeric_limits<double>::min())
```

Comment:
should not be positive value?

Reply:
`d_radiusFocal.getValue() >= std::numeric_limits<double>::min()` is equivalent to `d_radiusFocal.getValue() > 0`, isn't it?

Reply:
From what I understand: https://en.cppreference.com/w/cpp/types/numeric_limits
type	lowest()	min()		max()
double	-1.79769e+308	2.22507e-308	1.79769e+308

Reply:
@alxbilger I suppose so 😗